scripts/hipChatClient.js

```diff
@@ -116,6 +116,18 @@ HipChatClient.prototype.run = function (source, params) {
   self.checkProperties(source, this.sourceProperties);
   self.checkProperties(params, this.paramProperties);
 
+  if (!source.room_id) {
+    var re = /((?:http|https):\/\/.*?)\/v2\/room\/(\d+)/;
```

Comment:
1. is the trailing "/" expected/required?

2. not a blocker, obv, but my sense is that for this case, `lastIndexOf()` + `substr` would be easier to maintain than a regular expression.